src/Resources/PurchaseOrderDeliveryResource.cs

```diff
@@ -57,5 +57,7 @@ public class PurchaseOrderDeliveryResource
         public string FilCancelled { get; set; }
 
         public decimal? QtyPassedForPayment { get; set; }
+
+        public EmployeeResource ConfirmedBy { get; set; }
```

Comment:
Was this used anywhere?

Reply:
Not so far no but it must be used somewhere.. no idea where